src/features/types/index.ts

```diff
@@ -49,6 +49,12 @@ export type TextTagOptions =
     | 'span'
     | 'cite'
     | 'label'
+    | 'h1'
+    | 'h2'
+    | 'h3'
+    | 'h4'
+    | 'h5'
+    | 'h6'
 
 export type TypographyTagOptions = HeadingTagOptions | TextTagOptions
```

Comment:
These are all covered in `HeadingTagOptions` type so no need to add it I guess.

Reply:
Is this added to address the SEO issue?